libraries/WiFi/src/ssl_drv.h

```diff
@@ -0,0 +1,33 @@
+/* Provide SSL/TLS functions to ESP32 with Arduino IDE
+ * Ported from Ameba8195 (no license specified) to ESP32 by Evandro Copercini - 2017 - Public domain
```

Comment:
Yes, you are right.

These files process the input to make a good call to mod_ssl.c

I will rewrite them based on WifiClientSecure of ESP8266 and make this lib compatible with ESP8266 arduino sketches.

Also I will move out of the core folder as @me-no-dev request.